src/__test__/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprCompute.test.jsx

```diff
@@ -93,7 +93,26 @@ const store = mockStore({
       },
     ],
   },
-});
+  differentialExpression: {
+    properties: {
+      data: [],
+      cellSets: {},
+      loading: false,
+      error: false,
+      total: 0,
+    },
+    comparison: {
+      type: 'between',
+      group: {
+        between: {
+          cellSet: null,
+          compareWith: null,
+          basis: null,
+        },
+      },
```

Comment:
is there any reason why we are saving the `cellSet`, `compareWith` and `basis` under `group` and then under `between` instead of just doing this for example:

```
type: 'between',
cellSet: null,
compareWith: null,
basis: null,
```

Reply:
So that both states of the views (`between` and `within`) can be saved, and so user can go back and forth and see their selected options still there